backend/sdk/vnscope/__init__.py

```
@@ -1,11 +1,14 @@
-from .core import filter, order, profile, history, price, market
+from .core import configure
```

Ruff (0.11.9)

`.core.configure` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)